core/tests/protractor_utils/PreferencesPage.js

```diff
@@ -145,31 +149,46 @@ var PreferencesPage = function() {
   // might be abbreviated), rather than the text on the popover that appears
   // when hovering over the tile.
   this.expectDisplayedFirstSubscriptionToBe = async function(name) {
+    await waitFor.visibilityOf(
+      subscriptions.first(), 'subscriptions taking too long to appear.');
```

Comment:
Can you change the error message to specify that the first element failed to appear? We might have to debug solely with this error message, so it needs to be unique